backend/pyproject.toml

```diff
@@ -49,6 +49,7 @@ uvicorn = { extras = ["standard"], version = "^0.23.2" }
 neo4j = "^5.14.0"
 openfoodfacts_taxonomy_parser = { path = "../parser", develop = true }
 python-multipart = "^0.0.6"
+apscheduler = "^3.10.4"
```

Comment:
Is there any reason not to use the existing integration: https://pypi.org/project/fastapi_scheduler/ ?

Reply:
This library is not an official one from `tiangolo` and also seems to need a PostgresSQL DB connection